docs/naming.md

```diff
@@ -1,86 +1,169 @@
-```
+## Imagery Dataset S3 Paths
 
-10m Satellite RGB imagery covering the North Island captured in 2023
+The imagery dataset S3 path is also constructed from similar metadata.
 
 ```
-s3://nz-imagery/new-zealand/north-island_2023_10m/rgb/2193/collection.json
+<region>/
+  [<geographic_description>|<region>][_<survey_number>?]_<start_year>[-<end_year>?]_<gsd>m/
+    <product>/
+      <crs>/
 ```
 
-### Characters
+which can be broken down as:
+
+- the main <region> that the dataset covers
+- then if it exists, the <geographic_description> is used, if not, <region> is repeated instead (this would be the case where the imagery dataset contains full coverage of the region)
+- if it exists, the <survey_number> is used
+- then <start_year> (using all four digits to indicate the year)
+- if the imagery dataset was captured over multiple years, include a hyphen and the <end_year> (using all four digits to indicate the year)
+- then <gsd> (which is always in metres)
+- then <product> as multiple products may be created from the same imagery survey
+- then <crs> as we may store the data in different coordinate reference systems for different purposes
+
+### S3 Path Restrictions
+
+#### Characters
 
 The path is restricted to a limited set of characters (a-z, A-Z, 0-9, -, \_ ) and no whitespace.
 
-### Macrons
+#### Macrons
 
-When a name contains macrons "Ōtorohanga" the macron is removed from the path but retained in the metadata's Title / Description.
+When a name contains macrons, for example "Ōtorohanga", the macron is removed from the path but retained in the STAC Collection's Title / Description.
 
-### Apostrophes
+#### Apostrophes
 
-Where a name contains an apostrophe "Hawke's Bay" the apostrophe is removed from the path but retained in the metadata's Title / Description.
+Where a name contains an apostrophe, for example "Hawke's Bay", the apostrophe is removed from the path but retained in the STAC Collection's Title / Description.
 
-### Path components
+## Title and S3 Path Components
 
-#### `region`
+### `crs`
 
-Is taken from a list of regions
+EPSG Code for the coordinate reference system of the imagery. Generally this is [`2193`](https://epsg.io/2193) as it is the primary projection for most of LINZ's imagery.
 
-- New Zealand `new-zealand`
-- Northland `northland`
+### `geographic_description`
+
+This is free text and at the imagery maintainer's discretion. A specific city or sub-region or event name may be used to help describe the imagery capture area. The [Gazetteer](https://gazetteer.linz.govt.nz/) is referenced to ensure official names with correct spelling are used. If the region has full coverage, then the geographic description can be empty and the region will be used instead.
+
+### `geospatial_category`
+
+A general categorisation of imagery held within our archive that includes the following possible values:
+
+- Aerial Photos `aerial-photos`
+- DEM `dem` (only used in nz-elevation bucket)
```

Comment:
Should it be consitent with what is documented in the [linz/elevation](https://github.com/linz/elevation/blob/37799dbb9e83d8247489d6429f2be954b9eab9db/docs/naming.md#geospatial_category) repository?
Either removing `nz-elevation` related categories here, or adding `nz-imagery` categories there.

Reply:
removed